src/librustc_platform_intrinsics/lib.rs

```diff
@@ -37,60 +36,41 @@ pub enum IntrinsicDef {
     Named(&'static str),
 }
 
+
+static VOID: Type = Type::Void;
+
 static I8: Type = Type::Integer(true, 8, 8);
 static I16: Type = Type::Integer(true, 16, 16);
 static I32: Type = Type::Integer(true, 32, 32);
 static I64: Type = Type::Integer(true, 64, 64);
-static U8: Type = Type::Integer(false, 8, 8);
-static U16: Type = Type::Integer(false, 16, 16);
-static U32: Type = Type::Integer(false, 32, 32);
-static U64: Type = Type::Integer(false, 64, 64);
```

Comment:
Why were these types removed? This is removing a number of unsigned simd intrinsics from tests below. I was just hazarding a guess that it was related to the test failure, I'm not really sure what's going on there.